lib/FittableLayout/FittableLayout.js

```diff
@@ -91,6 +95,16 @@ var FittableLayout = module.exports = kind(/** @lends module:layout/FittableLayo
 	/**
 	* @private
 	*/
+	_reflowOnShowing: function(was) {
+		if (this.container.showing && this._hiddenReflow) {
```

Comment:
As a small simplification, could we use the current value (i.e. the second parameter passed to this observer), rather than access `this.container.showing`?